app/api/helpers/ticketing.py

```diff
@@ -1,7 +1,5 @@
 from datetime import datetime
 
-from flask_jwt import current_identity as current_user
```

Comment:
@iamareebjamal This was an oversight right? I think this is causing a 500 which is causing #6284

Reply:
Current_user became undefined after this.

Reply:
Yeah, my mistake. I thought this was covered by tests and did a Find/Replace. Not know how it slipped